src/modules/Dashboard/features/Applet/Schedule/ImportSchedulePopup/ImportSchedulePopup.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { FileUploader, Modal, SubmitBtnColor } from 'shared/components';
+import { ImportedFile } from 'shared/components/FileUploader/FileUploader.types';
```

Comment:
pls make export FileUploader.types in index and move import of the ImportedFile to from 'shared/components'?